src/pages/Home/Home.jsx

```diff
@@ -10,17 +10,28 @@ import {
   BottomNav,
 } from "components";
 import { useSelector, useDispatch } from "react-redux";
-import { getAllPosts } from "firebaseMethods";
-import { useEffect } from "react";
+
 
 export const Home = () => {
   const { allPosts } = useSelector((store) => store.allPosts);
+  const { user } = useSelector((store) => store.user);
+ 
+
+  const filterPosts = allPosts?.filter((p) =>
+    user?.following?.some((person) => person.userId === p?.data?.userId)
+  );
 
-  const dispatch = useDispatch();
+  // const filterByDate = [...filterPosts].sort(
+  //   (a, b) =>
+  //     new Date(a.data?.timeStamp.seconds * 1000) -
+  //     new Date(b.data?.timeStamp.seconds * 1000)
+  // );
+
+  // const isUndefined = (o) => typeof o.popularity === "undefined";
+  // const filterByLikes = filterPosts?.sort(
+  //   (a, b) => b?.data?.likes?.length - a?.data?.likes?.length
+  // );
```

Comment:
Remove comments if unnecessary

Reply:
Thanks for pointing it out, will remove it